packages/forms/backend/sirius-components-widget-reference-graphql/pom.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ Copyright (c) 2023 Obeo.
+ This program and the accompanying materials
+ are made available under the terms of the Eclipse Public License v2.0
+ which accompanies this distribution, and is available at
+ https://www.eclipse.org/legal/epl-2.0/
+
+ SPDX-License-Identifier: EPL-2.0
+
+ Contributors:
+     Obeo - initial API and implementation
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>3.1.2</version>
+		<relativePath /> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>org.eclipse.sirius</groupId>
+	<artifactId>sirius-components-widget-reference-graphql</artifactId>
+	<version>2023.8.4</version>
+	<name>sirius-components-widget-reference-graphql</name>
+	<description>Sirius Components Reference Wdiget GraphQL</description>
```

Comment:
Small typo here `Wdiget` -> `Widget`

The separation of the GraphQL API from the original project is a first step in the right direction but the dependencies of the original project `sirius-components-widget-reference`(not that one) are all over the place. 

Why does it still depend on:
- `sirius-components-graphql-api`?
- or `sirius-components-collaborative-trees`?
- or `sirius-components-widget-reference-view-edit`?
- or `sirius-components-representations`? (forms already brings this dependency)
- or `sirius-components-annotations`?

as for this pom.xml, why does it depend on:

- `graphql-java`? since `sirius-components-graphql-api` already bring this dependency
- `sirius-components-widget-reference`? since `sirius-components-collaborative-widget-reference` already brings this dependency

and I have not looked in details at all the dependencies involved.

Reply:
I've clean up pom.xml files